src/language-service/src/schemas/core.ts

```diff
@@ -0,0 +1,168 @@
+   * https://www.home-assistant.io/docs/configuration/basic/#longitude
+   *
+   * @minimum -180
+   * @maximum 180
+   */
+  longitude?: number;
+
+  /**
+   * Name of the location where Home Assistant is running.
+   * https://www.home-assistant.io/docs/configuration/basic/#name
+   */
+  name?: string;
+
+  /**
+   * Packages in Home Assistant provide a way to bundle different component’s configuration together. It allows for "splitting" your configuration.
+   * https://www.home-assistant.io/docs/configuration/packages/
+   */
+  packages?: Array<HomeAssistantRoot> | IncludeNamed;
+
+  /**
+   * Pick your time zone from the column TZ of Wikipedia’s list of tz database time
+   * https://www.home-assistant.io/docs/configuration/basic/#time_zone
+   * https://www.wikiwand.com/en/List_of_tz_database_time_zones
+   */
+  time_zone?: TimeZone;
+
+  /**
+   * "metric" for Metric, "imperial" for Imperial.
+   * This also sets temperature unit Home Assistant will use.
+   * https://www.home-assistant.io/docs/configuration/basic/#unit_system
+   */
+  unit_system?: UnitSystem;
+
+  /**
+   * List of folders that can be used as sources for sending files.
+   * https://www.home-assistant.io/docs/configuration/basic/#whitelist_external_dirs
+   */
+  whitelist_external_dirs?: string[];
+}
+
+export interface CoreCustomize {
+  [key: string]: CoreCustomizeItem | IncludeNamed;
+}
+
+/**
+ * @TJS-additionalProperties true
+ */
+interface CoreCustomizeItem {
+  /**
+   * For switches with an assumed state two buttons are shown (turn off, turn on) instead of a switch.
+   * https://www.home-assistant.io/docs/configuration/customizing-devices/#assumed_state
+   */
+  assumed_state?: boolean;
+
+  /**
+   * Sets the class of the device, changing the device state and icon that is displayed on the UI (see below).
+   * Please note: It does not set the unit_of_measurement.
+   * https://www.home-assistant.io/docs/configuration/customizing-devices/#device_class
+   */
+  device_class?: DeviceClasses;
+
+  /**
+   * URL to use as picture for entity.
+   * https://www.home-assistant.io/docs/configuration/customizing-devices/#entity_picture
+   */
+  entity_picture?: string;
+
+  /**
+   * Name of the entity as displayed in the UI.
+   * Please note that most of the time you can just rename the entity in the UI itself.
+   * https://www.home-assistant.io/docs/configuration/customizing-devices/#friendly_name
+   */
+  friendly_name?: string;
+
+  /**
+   * Any icon from MaterialDesignIcons.com. Prefix name with mdi:. For example: mdi:home.
+   * Please note that most of the time you can just change the entity icon in the UI itself.
+   * https://www.home-assistant.io/docs/configuration/customizing-devices/#icon
+   */
+  icon?: string;
+
+  /**
+   * Sets the initial state for automations, on (true) or off (false).
+   * https://www.home-assistant.io/docs/configuration/customizing-devices/#initial_state
+   */
+  initial_state?: boolean;
```

Comment:
Question on booleans, should we also allow for 'on' (with quotes)? It works but would require us to use boolean | string everywhere...

Reply:
We should not, as it is not according to YAML 1.2 specifications.
Furthermore, it increases the risks of issues.

Reply:
I Agree!